Orange/widgets/data/owsql.py

```diff
@@ -50,12 +51,15 @@ class OWSql(OWWidget):
     want_main_area = False
     resizing_enabled = False
 
+    cm_sql_username = CredentialManager("SQL username")
+    cm_sql_password = CredentialManager("SQL password")
```

Comment:
Could this be, as per CredentialManager's docstring, a single `cm = CredentialManager('SQL Table')` line, further below used as `self.cm.username`, `self.cm.password`?